configuration.h

```diff
@@ -476,6 +476,23 @@ typedef struct settings
       char input_android_physical_keyboard[255];
 #endif
 
+      char input_player1_reserved_device[255];
```

Comment:
I think instead this should be turned into a 3D array instead, and instead of a hardcoded '16 players', go with a define like MAX_USERS instead for the first array dimension.

So it'd be like

char input_players_reserved_device[MAX_USERS][255];

Reply:
Ooh, great idea 👍